[10.0] [BACKPORT] l10n_ch_qr_bill_scan (#561)
OCA/l10n-switzerland: l10n_ch_qr_bill_scan/readme/DESCRIPTION.rst

Import Vendor bills with a PayEye scanner.

> The QR bills are introduced in Switzerland since the 30th of June 2020.

Reviewer: ```suggestion
The QR bills were introduced in Switzerland on the 30th of June 2020.
```
